xpdacq/xpdacq.py

```diff
@@ -287,54 +329,45 @@ def dark(sample, scanplan, **kwargs):
     _yamify_dark(dark_def)
     return dark_uid
 
-def dryrun(sample,scan,**kwargs):
-    '''same as run but scans are not executed.
+def background(sample, scanplan, **kwargs):
```

Comment:
The function name and the docstring do not seem to agree with each other.  name suggests a background scan but docstring a dryrun